modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -134,7 +136,7 @@ export const createNodeQueryString = (
 export const createNetworkQuery = (
 	config: NodeConfig,
 	requestedFields: RequestedFieldsMap,
-): DocumentNode => {
+): DocumentNode | undefined => {
```

Comment:
Since this may return undefined, and this funciton is exported for use by other modules, it would be good to include a description in the TSDoc about when this will return `undefined` so other devs understand what this response is communicating.